Hackney.Shared.Tenure/Domain/TenureInformation.cs

```diff
@@ -31,5 +31,18 @@ public class TenureInformation
         public bool? IsSublet { get; set; }
         public DateTime? SubletEndDate { get; set; }
         public int? VersionNumber { get; set; }
+        public string JigsawRef { get; set; }
+        public string FundingSource { get; set; }
+        public int? NoOfAdults { get; set; }
+        public int? NoOfChildren { get; set; }
+        public bool? HasStorage { get; set; }
+        public bool? RentAccountRequired { get; set; }
+        public string NoRentAccountReason { get; set; }
+        public DateTime? ActualStartOfTenure { get; set; }
```

Comment:
This should be startOfTenureDate (already exists as a property). I have raised it with Mathew to confirm if the Reservation date is actually referring to the date of reservation between Hackney and the supplier and not the actual booking, in which case it should be stored in the contracts API and this won't be needed.